pg-client.cabal

```diff
@@ -53,29 +53,30 @@ library
   build-depends:
```

Comment:
Added explicit lower bounds, in anticipation of us publishing the library to Hackage.